addOns/pscanrulesAlpha/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Ignore CSS, JavaScript, images, or font files when scanning for source code disclosures (Issues: 6595, 6795, & 6820).
 - Update minimum ZAP version to 2.11.0.
 
+## [35] - 2021-11-07
+### Fixed
+- Add info alerts for each scripts found along with the integrity hash in subresource intergrity attribute scan rule (Issue 5894).
+
```

Comment:
You'll need to rebase this PR because this is quite behind the current state of things. The change note should be in a `### Added` header in the Unreleased section.

Reply:
Assuming `origin` is your fork, and `upstream` is `zaproxy/zap-extensions`, then something like:
```
git fetch upstream/main
git rebase upstream/main
```
At this point CHANGELOG.md will probably be in conflict. You'll need to move the note to the unreleased section as mentioned earlier. Then you can add it and continue the rebase.
```
git add CHANGELOG.md
git rebase --continue
```
(Might have to use the full path for the CHANGELOG file.)

You'll probably have to force push it back to origin at this point.
```
git push --force
```